utils/unittest/execution_state.go

```diff
@@ -24,7 +24,7 @@ const ServiceAccountPrivateKeySignAlgo = crypto.ECDSAP256
 const ServiceAccountPrivateKeyHashAlgo = hash.SHA2_256
 
 // Pre-calculated state commitment with root account with the above private key
-const GenesisStateCommitmentHex = "0a05e884b52abd732a09a917b24138dffc1f83d1e92f9f6e2f2fb115381292af"
```

Comment:
expected?

Reply:
I think its expected considering the cadence version update

Reply:
we might need to include this PR first https://github.com/onflow/flow-go/pull/1648 to see if it would change this hash